features/dd-sdk-android-session-replay/api/dd-sdk-android-session-replay.api

```diff
@@ -53,7 +54,6 @@ public final class com/datadog/android/sessionreplay/SessionReplayConfiguration
 }
 
 public final class com/datadog/android/sessionreplay/SessionReplayConfiguration$Builder {
-	public fun <init> ()V
```

Comment:
should this be re-generated? I guess no-arg constructor is still there (because of the use of default argument value)?

Reply:
In continuation of what we discussed, I've added an explicit no-args constructor